.github/workflows/zips.yml

```
@@ -208,3 +214,27 @@ jobs:
           comment-id: ${{ steps.find-comment.outputs.comment-id }}
           edit-mode: replace
           body: Build files for ${{ github.event.pull_request.head.sha }} have been deleted.
+
+  publish-to-wporg:
+    name: Publish to WordPress.org
+    runs-on: ubuntu-latest
+    environment: wordpress-plugin-svn
+    if: github.event_name == 'workflow_dispatch' && ( github.ref_type == 'tag' || inputs.release_version != '' )
+    needs: build-zips
+    timeout-minutes: 20
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: zip-files
+          path: ${{ github.ref }}
+      - name: Extract
+        run: |
+          unzip google-site-kit.zip
+          mv google-site-kit/* .
+          rm -rf google-site-kit *.zip
```

I think this is the line that is failing the glob check
```suggestion
          rm -rf google-site-kit ./*.zip
```

---

Updated.